src/contracts/pods/EigenPod.sol

```diff
@@ -761,24 +671,12 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         return sha256(abi.encodePacked(validatorPubkey, bytes16(0)));
     }
 
-    /**
-     * Calculates delta between two share amounts and returns as an int256
-     */
-    function _calculateSharesDelta(uint64 newAmountGwei, uint64 previousAmountGwei) internal pure returns (int256) {
+    /// @dev Calculates the delta between two Gwei amounts, converts to Wei, and returns as an int256
```

Comment:
nit: this comment says it converts to wei, but the function does not. (amounts actually remain in gwei / at whatever magnitude they were previously)

Reply:
[a644d48](https://github.com/Layr-Labs/eigenlayer-contracts/pull/515/commits/a644d481384069337c5582a3a779c69e90393eac)